orchagent/aclorch.cpp

```diff
@@ -2055,6 +2055,11 @@ bool AclRuleMirror::activate()
 
     if (!state)
     {
+        // if the mirror session is inactive, do not create the counter yet. FlexCounter registration will be skipped as well.
```

Comment:
In this situation, where the mirror session is inactive, was the ACL rule created and then removed?

Reply:
If the ACL rule was never created, is there a way to not create the counter at the beginning?
(Instead of it being created and then removed)

Reply:
Looking into it. Thanks for the suggestion.

Reply:
PR has been updated based on your comment. Thank you!

Reply:
Wonderful! :)